convex/schema.ts

```diff
@@ -9,6 +9,22 @@ export const notesTableSchema = v.object({
   tags: v.array(v.string()),
   title: v.string(),
   fulltext: v.string(),
+  userId: v.string(),
+  numberOfLikes: v.string()
+})
+
+export const userTableSchema = v.object({
+  userId: v.string(),
```

Comment:
We don't need a userId field in the table as convex already has a field called "_id" for every entry in a table.